python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -537,7 +537,8 @@ def power(attrs, inputs, proto_obj):
     if 'broadcast' in attrs and attrs['broadcast'] == 1:
         new_attrs = translation_utils._remove_attributes(new_attrs, ['broadcast'])
         return 'broadcast_power', new_attrs, inputs
-    return 'pow', new_attrs, inputs
+    mxnet_op = symbol.pow(inputs[0], inputs[1])
```

Comment:
why this change? you probably made this change to accommodate op_set version 7 of ONNX. will we support op_set versions older than 7?

Reply:
Reverting back to the older commit. Will figure out supporting different op_set versions and work on the Power operator separately.